cm-gateway/common.tfvars

```diff
@@ -32,3 +22,20 @@ issuer_name = "gateway-issuer"
 
 # Hostname that demo gateway to use. Default is "example.com"
 gateway_hostname = "example.com"
+
+# Name of testing gateway. Default is "cmsolver"
+gateway_name = "cmsolver"
+
+
+# The following parameters must be used in combination
+# 1. issuer_type is "SelfSigned" and envoy.service.type is "NodePort", which can be tested in a local kind environment.
+# 2. issuer_type is "ACME" and envoy.service.type is "LoadBalancer", which can be tested in a cloud environment such as GKE.
+
+# Type of Issuer that is used to issue certificates. Default is "SelfSigned", the other is "ACME"
+issuer_type = "ACME"
+
+# The arguments will be passed to the contour helm values
+contour_arguments = {
+  # The service type of envoy. Default is "NodePort", the other is "LoadBalancer"
+  "envoy.service.type" = "LoadBalancer"
```

Comment:
Suggestion: add a commented out line that suggests the user how to provide a static load balancer IP (so that they can configure DNS records before applying this module)

i.e `# "envoy.service.loadBalancerIP" = "<my-static-ip>"`